stack/build.Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:noble
+
+ARG sources
+ARG packages
+ARG package_args='--no-install-recommends'
+
+RUN echo "$sources" > /etc/apt/sources.list.d/ubuntu.sources
+
+RUN echo "debconf debconf/frontend select noninteractive" | debconf-set-selections && \
+  export DEBIAN_FRONTEND=noninteractive && \
+  apt-get -y $package_args update && \
+  apt-get -y $package_args upgrade && \
+  apt-get -y $package_args install locales && \
+  locale-gen en_US.UTF-8 && \
+  update-locale LANG=en_US.UTF-8 LANGUAGE=en_US.UTF-8 LC_ALL=en_US.UTF-8 && \
+  apt-get -y $package_args install $packages && \
+  rm -rf /var/lib/apt/lists/* /tmp/*
+
+RUN for path in /workspace /workspace/source-ws /workspace/source; do git config --system --add safe.directory "${path}"; done
+
+RUN curl -sL -o /usr/local/bin/yj https://github.com/sclevine/yj/releases/latest/download/yj-linux-amd64 \
+  && chmod +x /usr/local/bin/yj
```

Comment:
Was there a particular motivation for including `yj` here? We removed it in Jammy Full a little while ago here - https://github.com/paketo-buildpacks/jammy-full-stack/pull/106. I'd say we should do the same here unless there's a glaring reason not to